packages/react-examples/.storybook/preview.js

```diff
@@ -57,7 +57,7 @@ function addCustomDecorators() {
    */
   const customDecorators = new Set();
 
-  if (['react-cards', 'react-checkbox', 'react-tabs', 'react-toggle'].includes(packageNamePlaceholder)) {
+  if (['react-cards', 'react-checkbox', 'react-tabs'].includes(packageNamePlaceholder)) {
```

Comment:
Bit of a side thing but this block actually should only run for react-cards now (it's not for converged packages)
```suggestion
  if (packageNamePlaceholder === 'react-cards') {
```